### PR TITLE
GIVCAMP-128 | Separate homepage and basic page; hero animation update; CTA update

### PR DESCRIPTION
**Pull request by yvonnetangsu**
# READY FOR REVIEW

# Summary
- Separate out homepage and basic page component on Storyblok
- Have homepage component pull in homepage specific fields in Storyblok
- hero animation update
- CTA update
- other misc tweaks

# Review By (Date)
- Retro

# Review Tasks

## Setup tasks and/or behavior to test

1. Go to preview
2. Check that in the hero, as you scroll down, the "On Purpose" text follows you down for a bit
3. See that the intro in the big bracket appears
4. See that the theme cards are still in the theme section
5. Look at code

# Associated Issues and/or People
- JIRA ticket(s) - GIVCAMP-128

### CODE REVIEW

File: src/components/Homepage/HomepageHero.tsx [hunk 1]
```diff
@@ -70,11 +81,11 @@ export const HomepageHero = () => {
...
       <m.div
-        animate={{ opacity: [0.5, 1, 1, 0] }}
-        transition={{ duration: 1.6, times: [0, 0.2, 0.9, 1], delay: 0.3 }}
-        className="su-cc su-opacity-50 su-absolute su-top-0 su-left-0 su-right-0 su-mb-0 su-pt-120 md:su-pt-216 2xl:su-pt-228 su-max-w-full"
+        animate={{ opacity: [1, 0] }}
+        transition={{ duration: 0.6, delay: 1.2 }}
+        className="su-cc su-absolute su-top-0 su-left-0 su-right-0 su-mb-0 su-pt-120 md:su-pt-216 2xl:su-pt-228 su-max-w-full"
       >
-        <Heading as="h1" size="f8" leading="none" font="druk" color="white">
+        <Heading as="h2" size="f8" leading="none" font="druk" color="white">
```
Comment (yvonnetangsu) [inline src/components/Homepage/HomepageHero.tsx:88]: I added a screen reader only H1 for the homepage, so this "How will we...." line becomes the h2

File: src/components/Storyblok/SbThemeCard.tsx [hunk 2]
```diff
@@ -5,24 +5,23 @@ import { SbVerticalCardProps } from './SbVerticalCard';
...
+export type SbThemeCardProps = Omit<SbVerticalCardProps, 'isSmallHeading'>;
 
 export const SbThemeCard = ({
   blok: {
     _uid,
     heading,
     headingLevel,
     body,
-    // TODO: seperate alt as separate field
-    image: { filename, focus, alt } = {},
```
Comment (yvonnetangsu) [inline src/components/Storyblok/SbThemeCard.tsx:17]: I don't think there's a need for alt text for the theme card other than the empty ""

File: src/components/Storyblok/SbVerticalCard.tsx [hunk 3]
```diff
@@ -53,10 +52,9 @@ export const SbVerticalCard = ({
     body={body}
     imageSrc={filename}
     imageFocus={focus}
-    alt={alt}
-    textColor={isDarkTheme ? 'white' : 'black'}
     tabColor={paletteAccentColors[value] as AccentBgColorType}
     ctaLabel={ctaLabel}
+    ctaSrText={ctaSrText}
```
Comment (yvonnetangsu) [inline src/components/Storyblok/SbVerticalCard.tsx:57]: Allow for CTA Sr Only text from Storyblok for the vertical and theme cards

File: src/pages/index.tsx [hunk 4]
```diff
@@ -1,31 +1,40 @@
...
-      {/* <CreateBloks blokSection={blok.hero} />
-      <CreateBloks blokSection={blok.content} /> */}
-      <ProgressStories bgImage={blok.progressBgImage.filename}>
+      <Intro text={blok.intro} />
+      <ThemeSection
+        themeCardDiscovery={blok.themeCardDiscovery}
+        themeCardCitizen={blok.themeCardCitizen}
+        themeCardAcceleration={blok.themeCardAcceleration}
+        themeCardPlanet={blok.themeCardPlanet}
+      />
```
Comment (yvonnetangsu) [inline src/pages/index.tsx:34]: Intro and theme cards are now passed from Storyblok this way

**Comment by yvonnetangsu** [comment]
@sherakama Would you like to be tagged on every Campaign PR or just the ones that I think you might be interested in? 😄 I know you have a lot on your plate so trying not to add noise also

**Comment by yvonnetangsu** [comment]
Going to merge this on so I can merge this into my next PR 😄

**Comment by sherakama** [comment]
@yvonnetangsu I'm a watcher/lurker on the repo. You can tag me for questions but otherwise, I will see and sometimes review them as it is.

**Comment by yvonnetangsu** [comment]
> @yvonnetangsu I'm a watcher/lurker on the repo. You can tag me for questions but otherwise, I will see and sometimes review them as it is.

Cool - that's what I suspected @sherakama  😄 I'll continue to tag @rebeccahongsf as the primary retro reviewer, and tag you when the particular PR has something new that I think you might want to look at 👀

**Pull request merged**